pmdarima/utils/wrapped.py

```diff
@@ -54,13 +56,60 @@ def caller(*args, **kwargs):
 
 
 @inheritdoc(parent=sm_acf)
-def acf(x, unbiased=False, nlags=40, qstat=False, fft=False,
-        alpha=None, missing='none'):
-    return sm_acf(x=x, unbiased=unbiased, nlags=nlags,
-                  qstat=qstat, fft=fft, alpha=alpha,
-                  missing=missing)
+def acf(x, unbiased=None, nlags=None, qstat=False, fft=True,
+        alpha=None, missing='none', adjusted=False):
+    kwargs = {
+        "x": x,
+        "nlags": nlags,
+        "qstat": qstat,
+        "fft": fft,
+        "alpha": alpha,
+        "missing": missing,
+    }
+
+    # Handle kwarg deprecation in statsmodels 0.13.0
+    if parse_version(statsmodels.__version__) >= parse_version("0.13.0"):
```

Comment:
Move this into a new function def in `pmdarima.compat.statsmodels`:

```python
def _use_sm_13():
    return parse_version(statsmodels.__version__) >= parse_version("0.13.0")
```

(name it whatever you want) and then you can mock this in a unit test.

```python
from unittest import mock

@pytest.mark.parametrize("useit", [True, False])
def test_using_sm13(useit):
    with mock.patch("pmdarima.compat.statsmodels._use_sm_13") as mock_sm:
        mock_sm.return_value = useit
        x = acf(x)
        ...
```

Reply:
Had the conversation in Slack, but putting here for posterity. This will require some thinking... An example is a Python 3.9 pipeline which installs `0.13.0` or higher. If it mocks `< 0.13.0` and uses the deprecated keywords, it will fail. In order to be able to test both deprecated and non-deprecated keywords, all builds would have to be on `0.12.X`, the last (and first, actually) version to support both the old and new keywords